packages/slate-editor/src/components/EditorBlock/EditorBlock.module.scss

```diff
@@ -4,6 +4,7 @@
 .outer {
     @include editor-v4-block($void: false);
     position: relative;
+    caret-color: transparent;
```

Comment:
This makes the cursor invisible in Image caption fields.

Reply:
Nice, you found this, thanks. I will fix it :)

Reply:
Fixed with 201aa6a7f1106dc21977d6e1c64cad7b37bf8d17